app/views/users/show.html.erb

```diff
@@ -153,6 +153,14 @@
               </dt>
               <dd class="list-inline-item"><%= @user.home_location_name %></dd>
             <% end %>
+            <% if @user.company&.strip.present? %>
+              <dt class="list-inline-item m-0 align-text-bottom">
+                <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-suitcase-lg-fill" viewBox="0 0 16 16">
+                  <path d="M7 0a2 2 0 0 0-2 2H1.5A1.5 1.5 0 0 0 0 3.5v9A1.5 1.5 0 0 0 1.5 14H2a.5.5 0 0 0 1 0h10a.5.5 0 0 0 1 0h.5a1.5 1.5 0 0 0 1.5-1.5v-9A1.5 1.5 0 0 0 14.5 2H11a2 2 0 0 0-2-2zM6 2a1 1 0 0 1 1-1h2a1 1 0 0 1 1 1zM3 13V3h1v10zm9 0V3h1v10z" />
+                </svg>
+              </dt>
+              <dd class="list-inline-item"><%= @user.company %></dd>
```

Comment:
> Maybe optimistically linking to a matching wiki page could help by giving the user a way to verify for themselves.

For what it’s worth, I think a verification system (whether based on the wiki or backlinks or whatever) can be separate from this PR. The existing reporting system can probably keep things from getting out of hand in the meantime. If we’re concerned about impersonation, we could add a tooltip to this box indicating that the company affiliation is self-reported.